TradingOnTalentURAP/turnover_and_skills.py

```diff
@@ -0,0 +1,586 @@
+import pandas as pd
+import numpy as np
+import math
+from scipy.stats import mstats
+from sklearn.linear_model import LinearRegression
+import statsmodels.api as sm
+from scipy import stats
+import statsmodels
+import ast
+
+
+# Turnover
+
+# Import turnover data
+
+current = pd.read_csv("/Users/jacquelinewood/Documents/URAP/fastwhitepaper_month_current.csv",sep='\t',header=None)
+join = pd.read_csv("/Users/jacquelinewood/Documents/URAP/fastwhitepaper_month_join.csv",sep='\t',header=None)
+leave = pd.read_csv("/Users/jacquelinewood/Documents/URAP/fastwhitepaper_month_leave.csv",sep='\t',header=None)
```

Comment:
Would be great to abstract this from your machine--using argparse in the same way as pointed to in the prior comments...